config/explore-right-wall.json

```diff
@@ -0,0 +1,13 @@
+{
+  "version": 2,
+  "robot": {
+    "modules": {
+      "app": {
+          "init": {
+            "right_wall": True
```

Comment:
This should be lowercase. See for example https://json-schema.org/understanding-json-schema/reference/boolean.html